events/janus_gelfevh.c

```diff
@@ -0,0 +1,476 @@
+/*! \file   janus_gelfevh.c
+ * \author Lorenzo Miniero <lorenzo@meetecho.com>
+ * \copyright GNU General Public License v3
+ * \brief  Janus GelfEventHandler plugin
+ * \details  This is a trivial event handler plugin for Janus, which is only
+ * there to showcase how you can handle an event coming from the Janus core
+ * or one of the plugins. This specific plugin forwards every event it receives
+ * to a web server via an UDP request, using libcurl.
+ *
+ * \ingroup eventhandlers
+ * \ref eventhandlers
+ */
+
+#include "eventhandler.h"
+
+#include <math.h>
+// #include <curl/curl.h>
```

Comment:
Is libcurl used or not? It's linked in the Makefile, and checked in configure.ac, but you're not including it here. As a side note, please don't use `//` for comments, but use `/*  */` instead (code style).

Reply:
no was not necessary since it would help only for tcp support, and it is an easy send srteam function